packages/core-typings/src/IRoom.ts

```diff
@@ -199,6 +214,8 @@ export interface IOmnichannelGenericRoom extends Omit<IRoom, 'default' | 'featur
 	livechatData: any;
 	queuedAt?: Date;
 
+	verificationStatus: RoomVerificationState;
```

Comment:
A small suggesion: Let's try to group all the fields we're adding for this feature and also include a comment mentioning that they are being used for this particular feature. This would go a long way in helping any beginers get started